RELEASES.md

```diff
@@ -1,3 +1,132 @@
+- [Stabilize `slice_flatten`](https://github.com/rust-lang/rust/pull/125561/)
+
+
+<a id="1.80-Cargo"></a>
+
+Cargo
+-----
+**relnotes**
+- [Stabilize `-Zcheck-cfg` as always enabled](https://github.com/rust-lang/cargo/pull/13571/)
+- [Warn, rather than fail publish, if a target is excluded](https://github.com/rust-lang/cargo/pull/13713/)
+- [Add special `check-cfg` lint config for the `unexpected_cfgs` lint](https://github.com/rust-lang/cargo/pull/13913/)
+- [Stabilize `cargo update --precise <yanked>`](https://github.com/rust-lang/cargo/pull/13974/)
+- [Build only the specified artifact library when multiple types are available](https://github.com/rust-lang/cargo/pull/13842/)
+- [Don't change file permissions on `Cargo.toml` when using `cargo add`](https://github.com/rust-lang/cargo/pull/13898/)
+- [Support using `cargo fix` on IPv6-only networks](https://github.com/rust-lang/cargo/pull/13907/)
+
+
+<a id="1.80-Rustdoc"></a>
+
+Rustdoc
+-----
+
+- [Allow searching for references](https://github.com/rust-lang/rust/pull/124148/)
+- [Stabilize `custom_code_classes_in_docs` feature](https://github.com/rust-lang/rust/pull/124577/)
+- [fix: In cross-crate scenarios show enum variants on type aliases of enums](https://github.com/rust-lang/rust/pull/125300/)
+
+
+<a id="1.80-Compatibility-Notes"></a>
+
+Compatibility Notes
+-------------------
+- [rustfmt estimates line lengths differently when using non-ascii characters](https://github.com/rust-lang/rustfmt/issues/6203)
+- [Type aliases are now handled correctly in orphan check](https://github.com/rust-lang/rust/pull/117164/)
+- [Allow instructing rustdoc to read from stdin via `-`](https://github.com/rust-lang/rust/pull/124611/)
+- [`std::env::{set_var, remove_var}` can no longer be converted to safe function pointers and no longer implement the `Fn` family of traits](https://github.com/rust-lang/rust/pull/124636)
+- [Warn (or error) when `Self` ctor from outer item is referenced in inner nested item](https://github.com/rust-lang/rust/pull/124187/)
+- [Turn non-structural-const-in-pattern lints into hard errors](https://github.com/rust-lang/rust/pull/124661/)
+- [Make `WHERE_CLAUSES_OBJECT_SAFETY` lint a regular object safety violation](https://github.com/rust-lang/rust/pull/125380/)
+- [Convert `proc_macro_back_compat` lint to an unconditional error.](https://github.com/rust-lang/rust/pull/125596/)
+- [Do not parse CVarArgs (`...`) as a type in trait bounds](https://github.com/rust-lang/rust/pull/125863/)
+
+<a id="1.80-Internal-Changes"></a>
+
+Internal Changes
+----------------
+
+These changes do not affect any public interfaces of Rust, but they represent
+significant improvements to the performance or internals of rustc and related
+tools.
+
+- Misc improvements to size of generated html by rustdoc e.g. [#124738](https://github.com/rust-lang/rust/pull/124738/) and #[#123734](https://github.com/rust-lang/rust/pull/123734/)
```

Comment:
Isn't the size of the generated HTML a "public interfaces of Rust"?

Reply:
not really, there's no public API for the size. just like most other internal improgmements, users do benefit from it

Reply:
I don't really see how a size file could be part of "public interfaces" but in any case:
 * The generated HTML is not stable (although it rarely changes).
 * The size of the HTML/JS is definitely not stable (it has improvements and regressions all the time when we make changes).

I hope it answers your question @Urgau.